Tests/UnitTests/Core/Seeding/StrawLineSeederTest.cpp

```diff
@@ -11,21 +11,108 @@
 #include <set>
 
 #include "StrawHitGeneratorHelper.hpp"
+#include "TFile.h"
+#include "TTree.h"
 
 constexpr auto logLvl = Acts::Logging::Level::INFO;
 constexpr std::size_t nEvents = 5;
 
-ACTS_LOCAL_LOGGER(getDefaultLogger("StrawLineFitterTest", logLvl));
+ACTS_LOCAL_LOGGER(getDefaultLogger("StrawLineSeederTest", logLvl));
+
+#define DECLARE_BRANCH(dTYPE, NAME) \
+  dTYPE NAME{};                     \
+  outTree->Branch(#NAME, &NAME);
 
 namespace ActsTests {
 
+void testSeeder(RandomEngine& engine, TFile& outFile) {
+  auto outTree = std::make_unique<TTree>("StrawSeederTree", "StrawSeederTree");
+
+  DECLARE_BRANCH(double, trueY0);
+  DECLARE_BRANCH(double, trueTheta);
+  DECLARE_BRANCH(std::size_t, nTruthStraws);
+  DECLARE_BRANCH(std::size_t, nTruthStrips);
+  DECLARE_BRANCH(std::vector<double>, recoY0);
+  DECLARE_BRANCH(std::vector<double>, recoTheta);
+  DECLARE_BRANCH(std::vector<std::size_t>, nStraws);
+  DECLARE_BRANCH(std::vector<std::size_t>, nStrips);
+  DECLARE_BRANCH(std::size_t, nSeeds);
+
+  using GenCfg_t = MeasurementGenerator::Config;
+  GenCfg_t genCfg{};
+  genCfg.twinStraw = false;
+  genCfg.createStrips = true;
+
+  CompositeSpacePointLineSeeder::Config seederCfg{};
+  seederCfg.busyLayerLimit = 20;
+  const CompositeSpacePointLineSeeder seeder{
+      seederCfg, getDefaultLogger("StrawLineSeederTest", logLvl)};
+
+  for (std::size_t evt = 0; evt < nEvents; ++evt) {
+    if (evt % 100 == 0)
+      ACTS_INFO("Generating event " << evt);
```

Comment:
```suggestion
    if (evt % 100 == 0) {
      ACTS_INFO("Generating event " << evt);
    }
```
linter will complain about this